sdk/monitor/monitor-query/src/logsQueryClient.ts

```diff
@@ -56,17 +52,14 @@ export class LogsQueryClient {
    * @param options - Options for the LogsClient.
    */
   constructor(tokenCredential: TokenCredential, options?: LogsQueryClientOptions) {
-    const authPolicy = bearerTokenAuthenticationPolicy(
-      tokenCredential,
-      options?.scopes ?? defaultMonitorScope
-    );
-
```

Comment:
You should keep AAD support, see https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/schemaregistry/schema-registry/src/schemaRegistryClient.ts#L65-L66 for how to do it with core v2.

Reply:
Oh did this drop the AAD support. I will fix this